var/spack/repos/builtin/packages/star/package.py

```diff
@@ -0,0 +1,44 @@
+##############################################################################
+# Copyright (c) 2013-2016, Lawrence Livermore National Security, LLC.
+# Produced at the Lawrence Livermore National Laboratory.
+#
+# This file is part of Spack.
+# Created by Todd Gamblin, tgamblin@llnl.gov, All rights reserved.
+# LLNL-CODE-647188
+#
+# For details, see https://github.com/llnl/spack
+# Please also see the LICENSE file for our notice and the LGPL.
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License (as
+# published by the Free Software Foundation) version 2.1, February 1999.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the IMPLIED WARRANTY OF
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the terms and
+# conditions of the GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this program; if not, write to the Free Software
+# Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
+##############################################################################
+from spack import *
+import shutil
+
+
+class Star(Package):
+    """STAR is an ultrafast universal RNA-seq aligner."""
+
+    homepage = "https://github.com/alexdobin/STAR"
+    url      = "https://github.com/alexdobin/STAR/archive/2.5.3a.tar.gz"
+
+    version('2.5.3a', 'baf8d1b62a50482cfa13acb7652dc391')
+
+    def install(self, spec, prefix):
+        source_directory = join_path(self.stage.source_path, 'source')
+
+        with working_dir(source_directory):
+            make('STAR', 'STARlong')
+            make('install', 'STAR', 'STARlong')
+            shutil.move("../bin/STAR", prefix)
+            shutil.move("../bin/STARlong", prefix)
```

Comment:
As you've discovered, the `Makefile` contains:
```make
.PHONY: install
install:
        'mv' STAR STARlong ../bin
```
This is pretty much useless. Don't bother calling `make install`, just copy the files using the `install()` command.